ViewControllers/NewsFeedViewController.m

```diff
@@ -64,24 +60,18 @@ - (void) updateToPersonalizedNews{
     {
         PFUser *currentUser = [PFUser currentUser];
         [keywords addObjectsFromArray:[currentUser valueForKey:@"StocksOfInterest"]];
-        NSString *key = [keywords lastObject];
-        NSLog(@"TICKERS: %@", key);
+        NSUInteger rnd = arc4random_uniform((uint32_t)[keywords count]);
+        NSString *key = [keywords objectAtIndex:rnd];
         if (keywords != nil){
             [[APIManager shared] fetchHeadlineNews:(NSString *) key completion:^(NSMutableArray *keywordArticles, NSError *error) {
-                if (keywordArticles) {
-                    //[self.newsArray addObjectsFromArray:keywordArticles]
-                    NSLog(@"Successfully loaded Headline News");
-                    for (News *dict in keywordArticles) {
-                        [self.newsArray insertObject:dict atIndex:0];
-                        NSLog(@": Updated News Descriptions: %@", dict.description);
+                if (keywordArticles.count > 0) {
+                    for (int i = 0; i < 3; i++){
```

Comment:
Can you explain why you are using 3 here, I assume it's because you only want the top 3 results? If so it would be preferred to use a constant here instead of just using 3 or at least having a comment explaining why.